lib/services/http-service.js

```diff
@@ -229,5 +238,42 @@ function httpServiceFactory(
         return promises;
     };
 
+    /**
+     * Creates http proxy
+     * @private
+     * @param {Object} httpProxy - proxy parameters
+     * @returns
+     */
+    function createHttpProxy(httpProxy){
+        try {
+            var localPath = httpProxy.localPath || '/',
```

Comment:
if we're already storing static files (say in the directory `common` and you add a configuration for a proxy with `localpath: /common`, which gets precedence?

I think it may be worthwhile always preceding the URL space for the south-bound API set with a unique word to limit potential namespace collisions (i.e. always prefix the URL on the southbound API with `proxy` or something).

If the remote proxy is running on a awkward port - say port 9090 - is that accommodated with the server setting? Any reason to have server and remote path as the totally separate variables they are currently?